src/Dialog/index.scss

```diff
@@ -3,16 +3,13 @@
 }
 
 .nds-shim--dark {
-  position: fixed;
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
+  position: absolute;
+  top: env(safe-area-inset-top, 0px);
+  right: env(safe-area-inset-right, 0px);
+  bottom: env(safe-area-inset-bottom, 0px);
+  left: env(safe-area-inset-left, 0px);
```

Comment:
The `env()` values are part of the viewport API. The `env()` function in CSS will fall back to the second argument if the first is undefined.

`safe-area-*` values correspond to the area of a mobile browser window **not** covered by chrome (e.g. keyboard)